pkg/controller/pxcrestore/controller.go

```diff
@@ -247,9 +247,9 @@ func (r *ReconcilePerconaXtraDBClusterRestore) getBackup(cr *api.PerconaXtraDBCl
 	if cr.Spec.BackupSource != nil {
 		return &api.PerconaXtraDBClusterBackup{
 			ObjectMeta: metav1.ObjectMeta{
-				Name:        cr.Name,
-				Namespace:   cr.Namespace,
-				ClusterName: cr.ClusterName,
+				Name:      cr.Name,
+				Namespace: cr.Namespace,
+				// ClusterName: cr.ClusterName,
```

Comment:
With updated `apimachinery` `ObjectMeta` does not have `ClusterName` anymore, is it ok for us to stop using it @egegunes

Reply:
well, I believe it was a mistake from the beginning. I think we can remove

Reply:
[a0b5ff2](https://github.com/percona/percona-xtradb-cluster-operator/pull/1238/commits/a0b5ff229c9b5071430795d069b8053b03fd2134)